prow/plugins/approve/approve_test.go

```diff
@@ -55,23 +55,8 @@ func TestPluginConfig(t *testing.T) {
 	}
 	pa.Set(np)
 
-	orgs := map[string]bool{}
-	repos := map[string]bool{}
-	for _, config := range pa.Config().Approve {
-		for _, entry := range config.Repos {
-			if strings.Contains(entry, "/") {
-				if repos[entry] {
-					t.Errorf("The repo %q is duplicated in the 'approve' plugin configuration.", entry)
-				}
-				repos[entry] = true
-			} else {
-				if orgs[entry] {
-					t.Errorf("The org %q is duplicated in the 'approve' plugin configuration.", entry)
-				}
-				orgs[entry] = true
-			}
-		}
-	}
+	// no need to check for duplicates as the ConfigTree uses maps
+	// so yaml validation should have already failed
```

Comment:
YAML validation hilariously does not fail but we only get the last time you added the entry so no need to check for dupes

Reply:
Ok, will update the comment...